UPGRADE-4.0.md

```diff
@@ -16,6 +16,26 @@ Form
 
  * Support for data objects that implements both `Traversable` and
    `ArrayAccess` in `ResizeFormListener::preSubmit` method has been removed.
+   
+ * Using callable strings as choice options in ChoiceType has been deprecated
+   in favor of `PropertyPath`. Use a "\Closure" instead
+    
+    Before:
+   
+    ```php
+    'choice_value' => new PropertyPath('range'),
+    'choice_label' => 'strtoupper',
+    ```
+ 
+    After:
+   
+    ```php
+    'choice_value' => 'range',
+    'choice_label' => function ($choice) {
+    return strtoupper($choice);
```

Comment:
You misunderstood the "four" space here mean four for this line only.

Now on got one extra space from line 23 to 37 and missing 3 here one 35 :p

Sorry, after that we're done, thank you for working on this PR :)